api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -62,13 +70,29 @@ trait ValidationErrorConverter {
       .getOrElse("")
   }
 
-  private def relevantFields(err: ValidationError): JsObject = {
+  private def relevantRow(err: ValidationError, vs: HmdaFileValidationState): HmdaFileRow = {
+    if (err.ts) vs.ts.get
+    else vs.lars.find(lar => lar.loan.id == err.errorId).get
+  }
```

Comment:
I think using `get` in production code is dangerous in general, since it can cause an exception if nothing is found. Can these expressions ever return something other than a `Some`?. If so, `getOrElse` should be used.